app/views/clients/_form.html.slim

```diff
@@ -1,52 +1,42 @@
 = simple_form_for(@client) do |f|
   = simple_error_notice f
-  = f.input :salutation,
-              as: :select,
-              collection: Client.salutation_collection,
-              include_blank: true
+  = f.input :salutation, as: :select, collection: Client::SALUTATIONS, include_blank: true
 
   = f.simple_fields_for :contact do |c|
     = render 'contacts/fields', f: c
 
   .row
-    .col-xs-12.col-md-6
-      = f.input :birth_year,
-                as: :select,
-                collection: Client.year_collection,
-                include_blank: true
+    .col-xs-12.col-md-6= f.input :birth_year, as: :select, collection: Client.year_collection,
+                                 include_blank: true
   .row
-    .col-xs-12.col-md-6
-      = f.input :nationality, as: :country
-    .col-xs-12.col-md-6
-      = f.input :permit,
-                collection: Client::PERMITS,
-                as: :radio_buttons,
-                wrapper: :vertical_radio_and_checkboxes
+    .col-xs-12.col-md-4= f.input :nationality, as: :country
+    .col-xs-12.col-md-4= f.input :entry_date
+    .col-xs-12.col-md-4= f.input :permit, collection: Client::PERMITS, as: :radio_buttons,
+                                  wrapper: :vertical_radio_and_checkboxes
```

Comment:
maybe change the checkboxes to horizontal and then add the entry date above the nationality?

Reply:
After the rebase the layout was different, so now again it is next to nationality, but now it looks good. What do you think?